api/src/services/__tests__/competenciesService.ts

```diff
@@ -0,0 +1,19 @@
+import { listCompetencies } from '../competenciesService';
+import { mockQuery } from '../mockDb';
+
+describe('listCompetencies', () => {
```

Comment:
Would it be better to change "listCompetencies" to "competenciesService"?  This test file should test other service functions in the future.

Reply:
it was initially competenciesService. David suggested following patterns in other files in naming the functions.

Reply:
The patterns in the other files are that there it a top level `describe` block with the name of the module, and then an inner `describe` block for each export from that module.

Reply:
Just made changes as suggested.  We added a top-level 'describe' block.